webapp/src/pages/OneTimePassword.js

```diff
@@ -54,7 +54,11 @@ const OneTimePassword = () => {
       .authVerify({ phone: phoneNumber, token: otp.join("") })
       .then((r) => {
         setUser(r.data);
-        navigate("/dashboard");
+        if (r.data.onboarded === false) {
```

Comment:
Use `!r.data.onboarded` rather than the explicit false comparison, we don't want to differentiate between the two.